presto-main/src/main/java/io/prestosql/util/LongLong2LongOpenCustomHashMap.java

```diff
@@ -563,68 +517,56 @@ public boolean containsValue(final long v)
             return true;
         }
         for (int i = n; i-- != 0; ) {
-            if (!((key[i]) == (0)) && ((value[i]) == (v))) {
+            if (!((key[i * 2]) == (0) && (key[i * 2 + 1]) == (0)) && ((value[i]) == (v))) {
                 return true;
             }
         }
         return false;
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-
-    public long getOrDefault(final long k, final long defaultValue)
+    public long getOrDefault(final long k1, final long k2, final long defaultValue)
     {
-        if ((strategy.equals((k), (0)))) {
+        if ((strategy.equals((k1), (k2), (0), (0)))) {
             return containsNullKey ? value[n] : defaultValue;
         }
         final long[] key = this.key;
         // The starting point.
-        int pos = (it.unimi.dsi.fastutil.HashCommon.mix(strategy.hashCode(k))) & mask;
-        long curr = key[pos];
-        if (((curr) == (0))) {
+        int pos = (it.unimi.dsi.fastutil.HashCommon.mix(strategy.hashCode(k1, k2))) & mask;
+        long curr1 = key[pos * 2];
+        long curr2 = key[pos * 2 + 1];
+        if (((curr1) == (0)) && ((curr2) == (0))) {
             return defaultValue;
         }
-        if ((strategy.equals((k), (curr)))) {
+        if ((strategy.equals((k1), (k2), (curr1), (curr2)))) {
             return value[pos];
         }
         // There's always an unused entry.
         while (true) {
             pos = (pos + 1) & mask;
-            curr = key[pos];
-            if (((curr) == (0))) {
+            curr1 = key[pos * 2];
+            curr2 = key[pos * 2 + 1];
+            if (((curr1) == (0)) && ((curr2) == (0))) {
                 return defaultValue;
             }
-            if ((strategy.equals((k), (curr)))) {
+            if ((strategy.equals((k1), (k2), (curr1), (curr2)))) {
                 return value[pos];
             }
         }
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public long putIfAbsent(final long k, final long v)
+    public long putIfAbsent(final long k1, final long k2, final long v)
     {
-        final int pos = find(k);
+        final int pos = find(k1, k2);
         if (pos >= 0) {
             return value[pos];
         }
-        insert(-pos - 1, k, v);
+        insert(-pos - 1, k1, k2, v);
         return defRetValue;
     }
 
-    /**
-     * {@inheritDoc}
```

Comment:
@dain, are we sure we want to remove these in the original fork? The reason I have them removed here is because this is the actual commit that actually removes the interface and override (and thus the doc linkage).